aten/src/ATen/mps/MPSStream.mm

```diff
@@ -104,6 +112,8 @@ @interface MPSGraphExecutionDescriptor ()
     [_commandBuffer waitUntilCompleted];
     [_commandBuffer release];
     _commandBuffer = nil;
+    // reset the accumulated resource sizes for command buffer
+    _commandBufferResourceSize = 0;
```

Comment:
I think for `_prevCommandBuffer` also we should reset that. I know commitAndContinue is disabled, but for completeness sake,

Reply:
The prevCommandBuffer isn't committing here (just waits).